ext/mysql2/extconf.rb

```diff
@@ -73,14 +74,8 @@ def asplode lib
   rpath_dir = libs
 else
   inc, lib = dir_config('mysql', '/usr/local')
-  libs = ['m', 'z', 'socket', 'nsl', 'mygcc']
```

Comment:
I can't put my finger on this, but I believe this is for compatibility with _ancient_ compilers. Since we have tests running against Ruby 1.8, I'm not discounting that...